packages/world/src/modules/snapsync/ts/index.ts

```diff
@@ -0,0 +1,6 @@
+import "@latticexyz/store/register";
```

Comment:
should the plugin code be in `src/ts/plugins`, or https://github.com/latticexyz/mud/pull/584? 
modules are more like things that can be created/used by plugins

Reply:
for now probably in `src/ts/plugins`, and later in its own package (or 584)

Reply:
is there another plugins dir i don't know about or do you just mean one here?

Reply:
currently there is no plugins dir at all, I just meant to add one in `packages/world/src/ts/plugins` (temporarily, until we separate it as its own plugin)